### ISSUE TITLE
UserWarning

### ISSUE THREAD

**Issue by cy69855522** [ncullen93/torchsample#89]
Thanks for such a wonderful packet,but I met a warning when I using it.::>_<::

Epoch 1/1: 100%|██████████| 5/5 [00:06<00:00,  1.04s/ batches, loss=13.5649]/content/drive/AI/trademark/torchsample/modules/module_trainer.py:653: UserWarning: volatile was removed and now has no effect. Use `with torch.no_grad():` instead.
  input_batch = Variable(inputs[batch_idx*batch_size:(batch_idx+1)*batch_size], volatile=volatile)
/content/drive/AI/trademark/torchsample/modules/module_trainer.py:654: UserWarning: volatile was removed and now has no effect. Use `with torch.no_grad():` instead.
  target_batch = Variable(targets[batch_idx*batch_size:(batch_idx+1)*batch_size], volatile=volatile, requires_grad=False)
/content/drive/AI/trademark/torchsample/modules/module_trainer.py:510: UserWarning: invalid index of a 0-dim tensor. This will be an error in PyTorch 0.5. Use tensor.item() to convert a 0-dim tensor to a Python number
  eval_logs['val_loss'] = (samples_seen*eval_logs['val_loss'] + loss.data[0]*batch_size) / (samples_seen+batch_size)

**Comment by cy69855522** [comment]
Epoch 1/1: : 6 batches [00:07,  1.52s/ batches, loss=[], val_loss=[]]               
Saved model 0
Epoch 1/1: : 6 batches [00:09,  1.89s/ batches, loss=[], val_loss=[]]               
Saved model 1
Epoch 1/1: : 6 batches [00:09,  1.85s/ batches, loss=[], val_loss=[]]               
Saved model 2
Epoch 1/1: : 6 batches [00:10,  1.81s/ batches, loss=[], val_loss=[]]               
Saved model 3
Epoch 1/1: : 6 batches [00:07,  1.44s/ batches, loss=[], val_loss=[]]

**Comment by 2015E8009361057** [comment]
for example:
`im = Variable(im, volatile=True)`
now, it should be
`with torch.no_grad():
    im = Variable(im)
`
then the userwarning will be gone.

UserWarning: invalid index of a 0-dim tensor. This will be an error in PyTorch 0.5. Use tensor.item() to convert a 0-dim tensor to a Python number
eval_logs['val_loss'] = (samples_seeneval_logs['val_loss'] + loss.data[0]*batch_size) / (samples_seen+batch_size)

then it should be 
eval_logs['val_loss'] = (samples_seeneval_logs['val_loss'] + loss.item()*batch_size) / (samples_seen+batch_size)
